rclcpp/src/rclcpp/executor.cpp

```diff
@@ -598,70 +599,136 @@ take_and_do_error_handling(
 void
 Executor::execute_subscription(rclcpp::SubscriptionBase::SharedPtr subscription)
 {
+  using rclcpp::dynamic_typesupport::DynamicMessage;
+
   rclcpp::MessageInfo message_info;
   message_info.get_rmw_message_info().from_intra_process = false;
 
-  if (subscription->is_serialized()) {
```

Comment:
We'll want to coordinate with @mjcarroll and @alsora here, as this could run into conflict with what they are doing.

Reply:
I'll wait for their responses!

Reply:
I don't think that anything here conflicts with the other executor changes.  The `execute_*` family of functions deals with how to actually dispatch the work, while what I have been working on has been how that work is queued up.

@alsora what do you think?